migrations/11_create_nfts_metadata.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE nfts_metadata (
+  contract VARCHAR COLLATE NOCASE,
+  chain_id INTEGER NOT NULL,
+  token_id VARCHAR NOT NULL,
+  name VARCHAR NOT NULL,
+  symbol VARCHAR NOT NULL,
+  uri VARCHAR NOT NULL,
```

Comment:
how about fetching & including the actual json metadata that comes from this URL?